assets/css/style.css

```diff
@@ -1,172 +1,55 @@
 * {
-    font-family: Arial, Helvetica, sans-serif;
+  font-family: Arial, Helvetica, sans-serif;
 }
 
-
 /*========================================Header Design========================================*/
 .header {
-    padding: 5px;
-    background-color: #ffffff;
-    color: #ffffff;
-    background-image: url("../images/background.jpg");
-    background-size: cover;
-    background-position: center;
+  padding: 5px;
+  background-color: #ffffff;
+  color: #ffffff;
+  background-image: url("../images/background.jpg");
+  background-size: cover;
+  background-position: center;
 }
-  
+
 .header h1 {
-    font-size: 80px;
-    background-color: rgb(49, 52, 56, 0.85);
-    width: fit-content;
-    padding-right: 25px;
-    padding-left: 10px;
+  font-size: 80px;
+  background-color: rgb(49, 52, 56, 0.85);
+  width: fit-content;
+  padding-right: 25px;
+  padding-left: 10px;
 }
 
-/*========================================Navigation Bar Design========================================*/
-nav {
-    display: flex;
-    justify-content: flex-end;
-    background-color: #2A5559;
-    
+/*========================================Side Nav Bar========================================*/
+.sidenav li > a {
+  color: white;
 }
-  
-  nav a {
-    color: #ffffff;
-    padding: 14px 20px;
-    text-decoration: none;
-    text-align: center;
-    font-size: 150%;
-  }
-
-  nav ul li {
-      list-style: none;
-      margin: 0 20px;
-      display: inline;
-      text-transform: uppercase;
-  }
-
-  nav ul li:hover a {
-      background:#FF8B42;
-  }
-/*========================================About me Section========================================*/
-.about-me {
-    display: flex;
-    flex-wrap: wrap;
-    flex-direction: row;
-    justify-content: flex-start;
+.sidenav li > a > i {
+  color: white !important;
 }
 
-.about-me img {
-    padding: 10px;
-    flex-shrink: 1;
-    max-width: 300px;
-    max-height: 200px;
+/*========================================About me Section========================================*/
+#picture {
```

Comment:
Wherever possible, try to use classes in place of ids so that if needs be you can reuse your styles across different components on a page